### DISCUSSION TITLE
Flat-viewer: graph evolution of a numeric value over time

### DISCUSSION THREAD

**Discussion by julien-c** [githubocto/flat-action#15] [Ideas & Feature Requests]
Not sure if this is the best place to open this, or if I'd better head to https://github.com/githubocto/flat-viewer

### 🧙  The killer feature for me would be to be able to graph the evolution of values over time (i.e. across commits)

i.e. I'm using a Flat action to scrape the number of "things" from an API. I want to graph number of "things" over time 📈 

Let me know if I can help!

**If you like this feature, please upvote ❤️ 💜 💚**

**Comment by RandomFractals** [comment]
see: https://github.com/githubocto/flat-viewer/issues/17#issuecomment-852998337
